sonar-db/src/main/java/org/sonar/db/permission/AuthorizationDao.java

```diff
@@ -117,6 +133,10 @@ public int countUsersWithGlobalPermissionExcludingUserPermission(DbSession dbSes
       });
   }
 
+  /**
+   * @deprecated it loads too many results and there's no functional need.
+   */
+  @Deprecated
   public Collection<String> selectAuthorizedRootProjectsUuids(DbSession dbSession, @Nullable Integer userId, String role) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)